backend/src/jobs.rs

```diff
@@ -1233,6 +1233,25 @@ pub enum JobPayload {
     RawFlow { value: FlowValue, path: Option<String> },
 }
 
+lazy_static::lazy_static! {
+    // TODO: these aren't synced, they should be moved into the queue abstraction once/if that happens.
+    static ref JOBS_TOTAL_PUSHED: prometheus::IntCounter = prometheus::register_int_counter!(
+        "jobs_total_pushed",
+        "Total number of jobs pushed to the queue."
+    )
+    .unwrap();
+    static ref JOBS_TOTAL_DELETED: prometheus::IntCounter = prometheus::register_int_counter!(
+        "jobs_total_deleted",
+        "Total number of jobs deleted from the queue."
+    )
+    .unwrap();
+    static ref JOBS_TOTAL_PULLED: prometheus::IntCounter = prometheus::register_int_counter!(
+        "jobs_total_pulled",
+        "Total number of jobs pulled from the queue."
+    )
```

Comment:
I'm not sure of the usefulness of "total" here, especially because if there are multiple server apis, you would need to aggregate them to actually get the total number